includes/admin/extensions/class-extension-manager.php

```diff
@@ -0,0 +1,489 @@
+<?php
+
+namespace EDD\Admin\Extensions;
+
+use \EDD\Admin\Pass_Manager;
+use \EDD\Admin\Extensions\ProductData;
+
+class Extension_Manager {
+
+	/**
+	 * All of the installed plugins on the site.
+	 *
+	 * @since 2.11.x
+	 * @var array
+	 */
+	public $all_plugins;
+
+	/**
+	 * The minimum pass ID required to install the extension.
+	 *
+	 * @since 2.11.x
+	 * @var int
+	 */
+	private $required_pass_id;
+
+	/**
+	 * Pass Manager class
+	 *
+	 * @var \EDD\Admin\Pass_Manager
```

Comment:
```suggestion
	 * @var Pass_Manager
```

Full namespace isn't necessary since you import it further up.